pixl_core/src/core/patient_queue/message.py

```diff
@@ -37,6 +37,7 @@ class Message:
     procedure_occurrence_id: int
     project_name: str
     extract_generated_timestamp: datetime
+    pseudo_patient_id: str
```

Comment:
I don't think we need the patient id in the message, we're not using it in the imaging api

Reply:
How to put `pseudo_patient_id` as an optional variable in the message as removing it is breaking various tests:

cli/tests/test_check_env.py::test_check_env PASSED                                                                                                                                     [  4%]
cli/tests/test_check_env.py::test_check_env_fails PASSED                                                                                                                               [  8%]
cli/tests/test_database.py::test_project_doesnt_exist ERROR                                                                                                                            [ 13%]
cli/tests/test_database.py::test_first_image_exported ERROR                                                                                                                            [ 17%]
cli/tests/test_database.py::test_new_extract_with_overlapping_images ERROR                                                                                                             [ 21%]
cli/tests/test_database.py::test_processed_images_for_project PASSED                                                                                                                   [ 26%]
cli/tests/test_docker_commands.py::test_pixl_up_works PASSED                                                                                                                           [ 30%]
cli/tests/test_docker_commands.py::test_pixl_down_works PASSED                                                                                                                         [ 34%]
cli/tests/test_docker_commands.py::test_pixl_down_warns_on_volumes PASSED                                                                                                              [ 39%]
cli/tests/test_io.py::test_message_from_csv_raises_for_malformed_input PASSED                                                                                                          [ 43%]
cli/tests/test_io.py::test_make_radiology_linker_table PASSED                                                                                                                          [ 47%]
cli/tests/test_message_processing.py::test_no_retry_if_none_exported ERROR                                                                                                             [ 52%]
cli/tests/test_message_processing.py::test_retry_with_image_exported_and_no_change ERROR                                                                                               [ 56%]
cli/tests/test_message_processing.py::test_retry_with_image_exported_and_no_change_multiple_projects ERROR                                                                             [ 60%]
cli/tests/test_messages_from_files.py::test_messages_from_csv FAILED                                                                                                                   [ 65%]
cli/tests/test_messages_from_files.py::test_messages_from_csv_with_participant_id FAILED                                                                                               [ 69%]
cli/tests/test_messages_from_files.py::test_messages_from_csv_multiple_projects FAILED                                                                                                 [ 73%]
cli/tests/test_messages_from_files.py::test_messages_from_parquet FAILED                                                                                                               [ 78%]
cli/tests/test_messages_from_files.py::test_batch_upload FAILED                                                                                                                        [ 82%]
cli/tests/test_messages_from_files.py::test_duplicate_upload FAILED                                                                                                                    [ 86%]
cli/tests/test_messages_from_files.py::test_participant_id_in_csv FAILED                                                                                                               [ 91%]
cli/tests/test_populate.py::test_populate_queue_parquet FAILED                                                                                                                         [ 95%]
cli/tests/test_populate.py::test_populate_queue_and_start FAILED                                                                                                                       [100%]

Reply:
I think we'll also need to alter the code not to expect to have the `Message` contain `pseudo_patient_id`. Does that make sense?